src/shared/components/MenuLine.tsx

```diff
@@ -0,0 +1,9 @@
+import * as React from "react";
+import {observer} from "mobx-react";
+
+@observer
+export default class MenuLine extends React.Component<> {
+    render() {
+        return <hr className="menuLine"></hr>;
```

Comment:
@ChrisWakefield i don't think we need the menuLine styling.  Plain <hr /> (which will get Bootstrap styling) should be fine.

Reply:
Following your suggestion.  (To me, the plain <hr/> is almost invisible.)